awxkit/awxkit/cli/format.py

```diff
@@ -11,30 +11,30 @@ def add_authentication_arguments(parser, env):
     auth = parser.add_argument_group('authentication')
     auth.add_argument(
         '--conf.host',
-        default=env.get('TOWER_HOST', 'https://127.0.0.1:443'),
+        default=env.get('CONTROLLER_HOST', 'https://127.0.0.1:443'),
```

Comment:
@AlanCoding you mentioned that we would still be accepting the old vars, and I see you are still going to lay them down in the injector for the credential. But here I see us ONLY looking for the new vars

Reply:
Let's cherry-pick [this commit](https://github.com/ansible/awx/commit/3c34242eeb2e144a46471c268dc4648aef69a395) onto this PR.  It checks for both the `TOWER_` and `CONTROLLER_` env vars and makes tests pass.

Reply:
I have made the required changes.